oidc-handler/core/pom.xml

```diff
@@ -0,0 +1,283 @@
+
+                        <goals>
+
+                            <goal>scr</goal>
+
+                        </goals>
+
+                        <configuration>
+
+                            <!-- Private service properties for all services. -->
+
+                            <properties>
+
+                                <service.vendor>Adobe</service.vendor>
+
+                            </properties>
+
+                        </configuration>
+
+                    </execution>
+
+                </executions>
+
+                <configuration>
+
+                    <outputDirectory>${project.build.directory}/classes</outputDirectory>
+
+                </configuration>
+
+                <dependencies>
+
+                    <dependency>
+
+                        <groupId>org.slf4j</groupId>
+
+                        <artifactId>slf4j-simple</artifactId>
+
+                        <version>1.5.11</version>
+
+                    </dependency>
+
+                </dependencies>
+
+            </plugin>
+
+
+
+            <!-- Apache Felix Bundle Plugin -->
+
+
+
+            <plugin>
+
+                <groupId>org.apache.felix</groupId>
+
+                <artifactId>maven-bundle-plugin</artifactId>
+
+                <version>2.5.3</version>
+
+                <inherited>true</inherited>
+
+            </plugin>
+
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.7.0</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>maven-sling-plugin</artifactId>
+                <version>2.3.4</version>
+                <configuration>
+                    <slingUrl>http://localhost:8080/system/console</slingUrl>
+                    <user>admin</user>
+                    <password>admin</password>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
+
+            <artifactId>org.apache.felix.scr</artifactId>
+
+            <version>2.0.10</version>
+
+            <scope>provided</scope>
+
+        </dependency>
+
+
+
+        <dependency>
+
+            <groupId>org.apache.felix</groupId>
```

Comment:
You should use the new OSGi annotations instead of the old ones from Apache Felix. These were created while we waited for an official solution from the OSGi alliance. 

The artifacts you can use (for instance) https://github.com/apache/sling-org-apache-sling-commons-html/blob/e9ffa3b09a170492258a8513ac0caaac8432caba/pom.xml#L106-L115